defrag/routes/__init__.py

```diff
@@ -0,0 +1,14 @@
+from defrag import app
+from defrag.routes import bugs, dispatcher, docs, organizer, docs, reddit, search, suggestions, twitter, wiki
+
+app.include_router(bugs.router)
```

Comment:
Can we make it so that we do not need this here?
The ideal way would be for the module to provide the path to their routes file, so we could import them all without hardcoding their name.
That way adding new modules does not require changing this file.